aries-site/src/pages/components/menu.mdx

```diff
@@ -45,9 +46,10 @@ An icon can be used on a menu item to provide additional context about the actio
 
 There are various places a Menu may be useful within an application. If the user is going to be selecting from a list of options to execute an action, Menu is a good component to use. The usage of labels and icons allow a menu to be versatile for different layout contexts.
 
-### Within a header
+### Within a header using a default Button
```

Comment:
```suggestion
### Within a header
```
This is just the default menu -- I feel like specifying "using a default Button" makes it seem like the user has to do something extra.

Reply:
thats fine I wasnt sure to leave it or try and combine what we have with what Keeth put in Figma